display/type340.c

```diff
@@ -732,6 +744,11 @@ character(int n, unsigned char c)
     case CHRESC:                        /* escape */
         return 1;
     }
+    /* account for non spacing characters */

```

Comment:
Here I think we should replace the case stmt and the new conditions below with flags stored in a 6th byte in the character table.  It would be cleaner, and likely more like how the actual hardware did things.....

Yeah, there are five available low order bits, but it would be neater to have all the flags in one place, unless there need to be more than 8!!!